### PR TITLE
Fix/esbuild sourcemap

### PR DESCRIPTION
**Pull request by sandeep-vedam**
Following are the changes that are done in this PR : 
1. Fixed the default behavior of esbuild for building sourcemap for both es5 and es6 version. For esbuild sourcemap refer the below link : https://esbuild.github.io/api/#sourcemap
2. Fixed default behavior of rollup for building sourcemap for es5 version.

### CODE REVIEW

File: src/configs/esbuild.es5.config.js [hunk 1]
```diff
@@ -101,7 +103,7 @@ module.exports = (folder, globalName) => {
     target: 'es5',
     mainFields: buildHelpers.getResolveConfigForBundlers(),
     outfile: `${folder}/appBundle.es5.js`,
-    sourcemap,
+    sourcemap: sourcemap,
```
Comment (michielvandergeest) [inline src/configs/esbuild.es5.config.js:106]: is this necessary?

**Review by michielvandergeest — APPROVED** [review]
LGTM

File: src/configs/esbuild.es5.config.js (hunk 1 above)
Comment (sandeep-vedam) [reply src/configs/esbuild.es5.config.js:106]: As per the documentation, it has to be like that. If we want we can revert this to original

Comment (michielvandergeest) [reply src/configs/esbuild.es5.config.js:106]: it should be the same according to JS syntax. And the original version is a bit shorter and cleaner, I think 😊

**Review by uguraslan — APPROVED** [review]
LGTM

**Pull request merged**